mackup/applications/transmission.cfg

```diff
@@ -4,3 +4,8 @@ name = Transmission
 [configuration_files]
 Library/Preferences/org.m0k.transmission.plist
 Library/Application Support/Transmission/blocklists
+.config/transmission/blocklists
+.config/transmission/resume
+.config/transmission/torrents
```

Comment:
```.config/transmission/blocklists``` is same as 
```Library/Application Support/Transmission/blocklists```.

There are text files in ```.config/transmission/resume``` but ```.config/transmission/torrents``` contains binary files. I just included both because in the new setup of ```transmission``` to resume I'll need to get the same torrent files. So, I included both folders.